src/lib/data/token.js

```diff
@@ -0,0 +1,14 @@
+export default class Token {
+  constructor ({ textType, idWord, word, beforeWord, afterWord, hasLineBreak }) {
+    this.textType = textType
+    this.idWord = idWord
+    this.word = word
+    this.beforeWord = beforeWord
+    this.afterWord = afterWord
+    this.hasLineBreak = hasLineBreak
+  }
+
+  get couldBeUsedForAlignment () {

```

Comment:
another very nitpicky request -- could we rename this to  something simpler like "isAlignable" ?

Reply:
let's rename